### PR TITLE
Remove datepickers on the actual job applications

### PR DESCRIPTION
**Pull request by werebus**
Second (of three) part to #484

### CODE REVIEW

File: app/views/application_submissions/show.haml [hunk 1]
```diff
@@ -32,7 +32,7 @@
           %i= render_markdown prompt
       - else
         %td=prompt
-      %td=response
+        %td=format_response response, data_type
```
Comment (dirschn) [inline app/views/application_submissions/show.haml:35]:
```suggestion
        %td= format_response response, data_type
```

**Review by dirschn — CHANGES_REQUESTED** [review]
Logging in as Justin and viewing "my" application gives me 
image

**Comment by werebus** [comment]
> Logging in as Justin and viewing "my" application gives me ...

Did you run the rake task?

**Comment by dirschn** [comment]
> Did you run the rake task?

Absolutely not I didn't

**Review by dirschn — CHANGES_REQUESTED** [review]
sorry missed this before

**Review by dirschn — APPROVED** [review]

**Pull request merged**